src/Pages/Landing/Landing.test.js

```diff
@@ -3,6 +3,10 @@ import Landing from './Landing';
 
 test('renders learn react link', () => {
   render(<Landing/>);
-  const linkElement = screen.getByTestId(/main-content/i);
-  expect(linkElement).toBeInTheDocument();
+  const header = screen.getByAltText(/LogoDevolution/i);
```

Comment:
This works but I would check if the main container of components exists. At parent component, you don't want to know details of child component. In the child component you then check the details.

For example in each component you could use <div test-id='<componentName>_main_container'>. Then you could always check if the component is rendered with similar check.